code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -68,13 +68,15 @@
 			var/blocked = 0
 			if(ishuman(A))
 				var/mob/living/carbon/human/H = A
-				if(H.check_shields(90, "the [name]", src, attack_type = THROWN_PROJECTILE_ATTACK))
+				if(H.check_shields(0, "the [name]", src, attack_type = THROWN_PROJECTILE_ATTACK))
 					blocked = 1
 			if(!blocked)
 				L.visible_message("<span class ='danger'>[src] pounces on [L]!</span>", "<span class ='userdanger'>[src] pounces on you!</span>")
 				L.Weaken(5)
 				sleep(2)//Runtime prevention (infinite bump() calls on hulks)
 				step_towards(src,L)
+			else
+				weakened = 2

```

Comment:
This is because aliens don't have CANWEAKEN flag in their status_flags.
Mobs have it by default : https://github.com/tgstation/tgstation/blob/master/code/modules/mob/mob_defines.dm#L124
But aliens override that : https://github.com/tgstation/tgstation/blob/master/code/modules/mob/living/carbon/alien/alien.dm#L27